Gemfile.lock

```diff
@@ -95,17 +86,11 @@ GEM
     cocoon (1.2.15)
     concurrent-ruby (1.1.6)
     connection_pool (2.2.5)
-    countries (1.2.5)
-      currencies (~> 0.4.2)
-      i18n_data (~> 0.7.0)
+    countries (3.1.0)
```

Comment:
... and we're back.

Reply:
some wacky things went on, I think because countries was a dependency of another gem (activevalidators). Both are removed by the end